pkgs/by-name/op/opensearch-benchmark/package.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, python3
+, fetchPypi
+}:
+let
+  python = python3.override {
```

Comment:
Why are you creating a `python` here and using `python3` around the code?

Reply:
I thought that would prevent some packages from rebuilding